docs/content/getting-started.md

```diff
@@ -134,3 +134,7 @@ Of course, customize the array based on the `@types/` packages you have installe
 ## Silencing warnings
 
 Like React, Primer React emits warnings to the JavaScript console under certain conditions, like using deprecated components or props. Similar to React, you can silence these warnings by setting the `NODE_ENV` environment variable to `production` during bundling.
+
+## Testing
+
+Testing your application with Primer React is no different than testing your application with any other React library. Depending on your test environment and the testing libraries you use, you may need polyfills. For example if you are using `jest`, it runs via Node runtime and using [JSDOM](https://github.com/jsdom/jsdom) as a DOM implementation, so you will need to mock some browser APIs. We have [helpers](https://github.com/primer/react/blob/main/src/utils/test-helpers.tsx) that you can utilize to mock some of these APIs.
```

Comment:
Would it be helpful to also  add a note/code snippet on how to include this file of test helpers in Jest or would that be better placed somewhere else in the project?

Reply:
I think it is a great point but I don't think we export the file? I imagined they could copy the mocks they needed from the test-helpers but it would be ideal to import the file. Maybe I am missing something here though.

Reply:
I feel like copying and pasting those mocks is a reasonable approach if we want to avoid increasing the API surface area of Primer React.

Reply:
One vote for exporting test-helpers, I think copy pasting is a secret trick that we tell people after they ask us why it broke 😅

Reply:
Oh sorry yes I just realised we export the file! 

Would below be accurate to document?
```
import "@primer/react/lib-esm/utils/test-helpers"; (For ESM)
import "@primer/react/lib/utils/test-helpers"; (For CommonJS)
```

Reply:
Thank you! Just pushed a commit